app/models/learner-group.js

```diff
@@ -19,6 +19,7 @@ export default DS.Model.extend({
     }
   ),
   instructorGroups: DS.hasMany('instructor-group', {async: true}),
+  offering: DS.belongsTo('offering', {async: true, inverse: 'learnerGroups'}),
```

Comment:
I removed this line along with others. The changes are local currently, not reflected on here yet.